test/common-tests/request-field-validation-api-tests/number-field-validation-api-tests.ts

```diff
@@ -70,6 +58,19 @@ export function withNumberFieldValidationApiTests<RequestBodyItem>({
       });
     }
 
+    it(`returns a 400 response if ${fieldName} is null`, async () => {
+      const requestWithNullField = { ...validRequestItem, [fieldNameSymbol]: null };
+
+      const { status, body } = await makeRequest([requestWithNullField]);
+
+      expect(status).toBe(400);
+      expect(body).toMatchObject({
+        error: 'Bad Request',
+        message: expect.arrayContaining([expectedNonEmptyFieldError]),
+        statusCode: 400,
+      });
+    });
+
```

Comment:
Same as above - should we not also add a test for the other case (i.e., 2xx response if fieldName is null when required is false?)

Reply:
@oscar-richardson-softwire, I originally was going to do this but realised that we always prevent `null` for numbers (regardless of if it's required or not) and there's already a test for this in the file